### ISSUE TITLE
How to view VLOG information

### ISSUE THREAD

**Issue by HangJie720** [pytorch/pytorch#57118]
How to use VLOG, which is same to  specify TF_CPP_MIN_VLOG_LEVEL variable in TensorFlow.

**Comment by ngimel** [comment]
No, not really, vitals has a much narrower scope. Pytorch doesn't provide information similar to VLOG in tensorflow.

**Comment by zou3519** [comment]
@HangJie720 could you tell us a bit more about your use case for VLOG?

**Comment by HangJie720** [comment]
for example in pytorch1.8.1/caffe2/operators/conv_op_cudnn.cc: line 262
VLOG(1) << "CUDNN Convolution: float16_compute specified and "
                << "supported, input data is Half - using Half "
                << "compute.";
how to run op test by calling torch.nn.conv2d() to print VLOG(1) message?

**Comment by ngimel** [comment]
caffe2 is deprecated.

**Comment by HangJie720** [comment]
@ngimel 
third_party/kineto/libkineto/src/output_json.cpp：line 354,
VLOG(2) << memcpy.correlationId << ": MEMCPY"; 

pytorch/c10/core/CPUAllocator.cpp
VLOG(1)
<< "There is not caching allocator registered for CPU, use the default allocator instead.";
these are not deprecated, how to print?

**Comment by veyron95** [comment]
Hi, @gchanan  I want to know how to print this logging information by running python programs. if need to enable Debug mode to compile or specify USE_GLOG option ON, or other solutions? 
I am the same as you need. @HangJie720

**Comment by ppwwyyxx** [comment]
Related: https://github.com/pytorch/pytorch/issues/135135
